src/rgw/rgw_lua.cc

```diff
@@ -133,40 +133,49 @@ int add_package(const DoutPrefixProvider *dpp, rgw::sal::Driver* driver, optiona
     return ret;
   }
 
-  auto lua_mgr = driver->get_lua_manager();
-
-  return lua_mgr->add_package(dpp, y, package_name);
+  return driver->get_lua_manager("")->add_package(dpp, y, package_name);
```

Comment:
i don't like that `LuaManager` will sometimes have a real path and sometimes a fake/empty one. maybe the functions that don't require a path should just be members of `sal::Driver` like `add_lua_package()` etc. then we don't need to allocate a `LuaManager` just to call one function

Reply:
agree, that makes much more sense (and save on unnecesary allocation)

Reply:
@cbodley @dang this is done here: a3acc32ad5940ed33774655f083156425680bcc6